backend/fms_core/models/project.py

```diff
@@ -0,0 +1,57 @@
+import reversion
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from .tracked_model import TrackedModel
+from django.contrib.auth.models import User
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._utils import add_error as _add_error
+from ._validators import name_validator, email_validator
+
+__all__ = ["Project"]
+
+
+@reversion.register()
+class Project(TrackedModel):
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH,
+                            help_text="The name of the project.",
+                            validators=[name_validator])
+
+    principal_investigator = models.CharField(blank=True, max_length=200, help_text="The principal investigator of the project.")
+
+    requestor_name = models.CharField(blank=True, max_length=200, help_text="The name of the requestor of the project.")
+
+    requestor_email = models.CharField(blank=True, max_length=200,
+                                       help_text="The email of the requestor of the project.",
+                                       validators=[email_validator])
+
+    targeted_end_date = models.DateTimeField(blank=True, help_text="Targeted date to conclude the project.")
+
+    IN_PROCESS = "In Process"
+    FINISHED = "Finished"
+
+    STATUS_CHOICES = (
```

Comment:
I recommend using a similar approach to how choices for value_type in PropertyType are declared. In that way, we do not declare variables for each status that are then not useful on their own, but again this is a personal preference :P However, when there is a list of 'variables', it is often recommended in Python to put them in a dictionary or a list.

Reply:
Ok, I used the approach used in individual, did not know the approach used in value_type was better. Changes are made ;)